spec/Appendix B -- Grammar Summary.md

```diff
@@ -69,18 +69,25 @@ ExponentIndicator :: one of `e` `E`
 Sign :: one of + -
 
 StringValue ::
-  - `""`
-  - `"` StringCharacter+ `"`
+  - `"` StringCharacter* `"`
+  - `"""` MultiLineStringCharacter* `"""`
 
 StringCharacter ::
   - SourceCharacter but not `"` or \ or LineTerminator
   - \u EscapedUnicode
   - \ EscapedCharacter
 
+MultiLineStringCharacter ::
+  - SourceCharacter but not `"""` or `\"""`
+  - `\"""`
+
 EscapedUnicode :: /[0-9A-Fa-f]{4}/
```

Comment:
Is it intended that MultilineStrings can contain escaped characters like \b or \u1234 and so on?

If so then I think this is spec incomplete.  It says that `MultilineCharacter` can be any Source character or it can be  `\"""`

But this definition if taking like that does not allow \uNNNN or \t or so on as way to encode characters that might not easily represented in source files.

I think (if the intent is to allow \ escaping that it should be

    MultiLineStringCharacter ::
       - SourceCharacter but not `"""` or `\"""`
       - `\"""`
       - \u EscapedUnicode
       - \ EscapedCharacter

If the intent is NOT to allow \x escaping in multi lines strings then I guess it would be correct.  But then I would disagree with that position, especially around unicode characters aka \uNNNN